locales/en/population.json

```diff
@@ -0,0 +1,5 @@
+{
+  "spaces_with_count": "{{count}} space",
+  "spaces_with_count_plural": "{{count}} spaces",
+  "add_space": "Add"
```

Comment:
Maybe `Add space` keeping inline with the _thing_ we're referring to? I always think about how links would be read out by a screenreader in instances like this. Without the visual context "Add" on its own wouldn't be very clear.

You could add it with a `visually-hidden` class if we really want to the button visually only say "Add" but it feels like it wouldn't have a negative impact if we add the word space?

Reply:
Hmm... I see what you mean, but it's not adding a space. On the subsequent daily summary screen, we have "Update numbers", so I guess it would be "Add numbers". That does start to affect the table layout as it's much wider than a few characters of numbers..

Also, thinking about screen readers, I'm honestly not sure what would be more useful for our end users. All of the "spaces" links go to the daily summary screen, which then allows you to update/add numbers. Are the "2 spaces" titles enough, or should they include the context of where they are? If the users are familiar with the system, is the word "Add" enough?

That's a lot of questions, but no actual answers :(

@Andy-Millar Any thoughts on this?
-

Reply:
@teneightfive @Andy-Millar Ok, how about I change this to say "Add numbers", but we'll review it again once we have this more integrated.

Reply:
Exactly this

Reply:
However 'Add numbers' will be weird once numbers have already be added. Action should be 'Change numbers' or 'Update numbers' in that case

Reply:
@Andy-Millar Oh, hang on - we're talking about the main table grid, not the daily summary page, e.g. the image above

Reply:
D'oh! Sorry! 'Add numbers' if fine then - if space is an issue we can fall back to 'Add'